zcash_primitives/src/transaction/components/sapling/builder.rs

```diff
@@ -225,6 +243,14 @@ impl<P: consensus::Parameters> SaplingBuilder<P> {
         }
     }
 
+    pub fn inputs(&self) -> &[impl SaplingInput] {
+        &self.spends
+    }
+
+    pub fn outputs(&self) -> &[SaplingOutput] {
```

Comment:
Does this need to be public? If so:
- Needs documentation.
- Document this addition in the changelog.